Dockerfile

```diff
@@ -6,13 +6,14 @@ ENV DEBIAN_FRONTEND noninteractive
 # install base packages
 RUN apt update && \
     apt install -y \
-      build-essential \
-      wget \
-      cmake \
-      libgtest-dev \
-      libgmock-dev \
-      net-tools \
-      git
+    build-essential \
+    wget \
+    cmake \
+    libgtest-dev \
+    libgmock-dev \
+    net-tools \
+    lcov \
+    git
```

Comment:
The purpose of the extra indent was for visual clarity that each of these lines is an argument to the command `apt install -y` rather than a new command. Additionally, by dedenting all these lines, it hides the real change (which is the addition of `lcov` to the list).

Unless there's a good reason we should remove the indentation, I think we should leave it as-is.

Reply:
I believe the indentation change was a side effect of the VS Code Docker plugin auto-formatting the file on save. I think it is okay to leave the indentation as it was, although this might become a recurring issue for contributors.

Reply:
I restored the indentation in this commit https://github.com/mit-dci/opencbdc-tx/pull/58/commits/a27f2100fa7da1ee4f27312440e93808a4b51b32 , although I had to disable the VS Code Docker Extension. You may run into this problem again with other contributors.